src/components/resizer.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import Canvas from './canvas.js';
-import Panel from './panel.js';
+import Panel from './scene.js';
```

Comment:
It will work but it's better to import files without extensions. And name components with PascalCase